jib-gradle-plugin/src/test/java/com/google/cloud/tools/jib/gradle/GradleProjectPropertiesTest.java

```diff
@@ -361,7 +362,9 @@ public void testCreateContainerBuilder_noClassesFiles() throws InvalidImageRefer
     Mockito.when(mockMainSourceSetOutput.getClassesDirs())
         .thenReturn(new TestFileCollection(ImmutableSet.of(nonexistentFile)));
     gradleProjectProperties.createContainerBuilder(
-        RegistryImage.named("base"), AbsoluteUnixPath.get("/anything"));
+        RegistryImage.named("base"),
+        AbsoluteUnixPath.get("/anything"),
+        ContainerizingMode.EXPLODED);
```

Comment:
Should we have a constant for the default mode?  If we change the default, then we need to remember to change this value here (and in setupBuildConfiguration) too